packages/jest-jasmine2/src/jasmine/Spec.ts

```diff
@@ -98,10 +98,10 @@ export default class Spec {
   static pendingSpecExceptionMessage: string;
 
   static isPendingSpecException(e: Error) {
-    return !!(
+    return Boolean(
```

Comment:
there is one eslint rule which imho would help with code blocks with lots of operators
`https://eslint.org/docs/rules/operator-linebreak`
so it would look like
```ts
e
&& e.toString
&& e.toString()...
```
having stuff at the start of the line is usually easier to keep cognitive complexity on the brain in place.
You can negotiate it with Simen if he wants it, it would be a good issue if you want to tackle it

Reply:
yeah sure let me just clear this one and then discuss with simen 😀

Reply:
`e.toString?.().includes(Spec.pendingSpecExceptionMessage)` or something probably, but just disable the rule in this (and other `packages/jest-jasmine2/src/jasmine/*.ts` files) - they're forked from jasmine and can remain a bit messy 🙂

Reply:
yes, in this and the other jasmine files